hw14-mvc/src/main/java/ru/otus/controllers/UserController.java

```diff
@@ -0,0 +1,46 @@
+package ru.otus.controllers;
+
+import java.util.List;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.view.RedirectView;
+
+import lombok.extern.slf4j.Slf4j;
+import ru.otus.model.User;
+import ru.otus.service.UserService;
+
+@Slf4j
+@Controller
+public class UserController {
+
+    private final UserService repository;
```

Comment:
```suggestion
    private final UserService userService;
```

Вроде сервис, не репозиторий.